docs/contribute.rst

```diff
@@ -84,6 +84,18 @@ everything needed for the tmt development ready on your system::
 
     pip install '.[all]'
 
+Install ``pre-commit`` to run all available checks before commit::
+
+    sudo dnf -y install pre-commit
+
+In case you prefer installation via ``pip``::
+
+    pip install pre-commit
+
+And install the ``pre-commit`` hooks::
+
+    pre-commit install
```

Comment:
I followed the instructions, and ran into the following warning:

> [WARNING] The 'rev' field of repo 'https://github.com/pre-commit/mirrors-autopep8' appears to be a mutable reference (moving tag / branch).  Mutable references are never updated after first install and are not supported.  See https://pre-commit.com/#using-the-latest-version-for-a-repository for more details.  Hint: `pre-commit autoupdate` often fixes this.

Shall we include `pre-commit autoupdate` in the instructions as well?

Reply:
I would rather use revision then